variables.tf

```diff
@@ -87,12 +81,10 @@ variable "cloud_services" {
         "hs-crypto",
         "hyperp-dbaas-mongodb",
         "hyperp-dbaas-postgresql",
-        "iam-identity",
         "iam-svcs",
         "is",
         "kms",
         "resource-controller",
-        "secrets-manager",
```

Comment:
secrets-manager is included the list in #396 at https://cloud.ibm.com/docs/vpc?topic=vpc-vpe-supported-services